### ISSUE TITLE
采用ch_ppocr_mobile_v2.0_rec_pre模型训练hwdb1.0数据集 100轮后acc开始在0.66左右波动 最终训练完的评估acc为0

### ISSUE THREAD

**Issue by WangJerry1229** [PaddlePaddle/PaddleOCR#4425]
初次尝试就从hwdb中抽取了三个字对应的1000多张数据集训练 之前采用相同的config训练了一个中文手写minst数据集效果还不错 结果将数据集更换为hwdb后就出现了这种情况 acc达到0.66后再增加轮数也没有效果 想问一下该如何解决

以下是配置文件：
Global:
  use_gpu: true
  epoch_num: 200
  log_smooth_window: 20
  print_batch_step: 5
  save_model_dir: ./output/hwdb
  save_epoch_step: 5
  eval_batch_step: [0, 2000]
  cal_metric_during_train: True
  pretrained_model: ./pretrain_models/ch_ppocr_mobile_v2.0_rec_pre/best_accuracy
  checkpoints: ./output/hwdb/best_accuracy
  save_inference_dir: ./output/hwdb/inference
  use_visualdl: False
  infer_img: 
  character_dict_path: ./dataset2/test_char.txt
  character_type: ch
  max_text_length: 25
  infer_mode: False
  use_space_char: True
  save_res_path: ./output/hwdb/hwdb_save.txt

Optimizer:
  name: Adam
  beta1: 0.9
  beta2: 0.999
  lr:
    name: Cosine
    learning_rate: 0.001
  regularizer:
    name: 'L2'
    factor: 0.00001

Architecture:
  model_type: rec
  algorithm: CRNN
  Transform:
  Backbone:
    name: MobileNetV3
    scale: 0.5
    model_name: small
    small_stride: [1, 2, 2, 2]
  Neck:
    name: SequenceEncoder
    encoder_type: rnn
    hidden_size: 48
  Head:
    name: CTCHead
    fc_decay: 0.00001

Loss:
  name: CTCLoss

PostProcess:
  name: CTCLabelDecode

Metric:
  name: RecMetric
  main_indicator: acc

Train:
  dataset:
    name: SimpleDataSet
    data_dir: ./dataset2
    label_file_list: ["./dataset2/train.txt"]
    transforms:
      - DecodeImage: # load image
          img_mode: BGR
          channel_first: False
      - RecAug: 
      - CTCLabelEncode: # Class handling label
      - RecResizeImg:
          image_shape: [3, 32, 320]
      - KeepKeys:
          keep_keys: ['image', 'label', 'length'] # dataloader will return list in this order
  loader:
    shuffle: True
    batch_size_per_card: 128
    drop_last: True
    num_workers: 8

Eval:
  dataset:
    name: SimpleDataSet
    data_dir: ./dataset2
    label_file_list: ["./dataset2/test.txt"]
    transforms:
      - DecodeImage: # load image
          img_mode: BGR
          channel_first: False
      - CTCLabelEncode: # Class handling label
      - RecResizeImg:
          image_shape: [3, 32, 320]
      - KeepKeys:
          keep_keys: ['image', 'label', 'length'] # dataloader will return list in this order
  loader:
    shuffle: False
    drop_last: False
    batch_size_per_card: 128
    num_workers: 8

训练日志（最后几次）：
[2021/10/23 08:07:45] root INFO: epoch: [190/200], iter: 1135, lr: 0.000092, loss: 0.359576, acc: 0.671875, norm_edit_dis: 0.671875, reader_cost: 0.18141 s, batch_cost: 0.28897 s, samples: 256, ips: 177.18275
[2021/10/23 08:07:48] root INFO: save model in ./output/hwdb\latest
[2021/10/23 08:07:48] root INFO: save model in ./output/hwdb\iter_epoch_190
[2021/10/23 08:07:48] root INFO: Initialize indexs of datasets:['./dataset2/train.txt']
[2021/10/23 08:07:49] root INFO: epoch: [191/200], iter: 1140, lr: 0.000089, loss: 0.363993, acc: 0.667969, norm_edit_dis: 0.667969, reader_cost: 0.09459 s, batch_cost: 0.14926 s, samples: 128, ips: 171.51357
[2021/10/23 08:07:53] root INFO: epoch: [191/200], iter: 1145, lr: 0.000086, loss: 0.363842, acc: 0.671875, norm_edit_dis: 0.671875, reader_cost: 0.38347 s, batch_cost: 0.66006 s, samples: 640, ips: 193.92211
[2021/10/23 08:07:54] root INFO: save model in ./output/hwdb\latest
[2021/10/23 08:07:54] root INFO: Initialize indexs of datasets:['./dataset2/train.txt']
[2021/10/23 08:07:58] root INFO: epoch: [192/200], iter: 1150, lr: 0.000082, loss: 0.363314, acc: 0.675781, norm_edit_dis: 0.675781, reader_cost: 0.41360 s, batch_cost: 0.69518 s, samples: 640, ips: 184.12502
[2021/10/23 08:07:59] root INFO: save model in ./output/hwdb\latest
[2021/10/23 08:07:59] root INFO: Initialize indexs of datasets:['./dataset2/train.txt']
[2021/10/23 08:08:03] root INFO: epoch: [193/200], iter: 1155, lr: 0.000079, loss: 0.367553, acc: 0.664062, norm_edit_dis: 0.664062, reader_cost: 0.33441 s, batch_cost: 0.56232 s, samples: 512, ips: 182.10355
[2021/10/23 08:08:05] root INFO: save model in ./output/hwdb\latest
[2021/10/23 08:08:05] root INFO: Initialize indexs of datasets:['./dataset2/train.txt']
[2021/10/23 08:08:07] root INFO: epoch: [194/200], iter: 1160, lr: 0.000076, loss: 0.365992, acc: 0.671875, norm_edit_dis: 0.671875, reader_cost: 0.24659 s, batch_cost: 0.41112 s, samples: 384, ips: 186.80706
[2021/10/23 08:08:10] root INFO: save model in ./output/hwdb\latest
[2021/10/23 08:08:10] root INFO: Initialize indexs of datasets:['./dataset2/train.txt']
[2021/10/23 08:08:12] root INFO: epoch: [195/200], iter: 1165, lr: 0.000073, loss: 0.366543, acc: 0.664062, norm_edit_dis: 0.664062, reader_cost: 0.18912 s, batch_cost: 0.30051 s, samples: 256, ips: 170.37474
[2021/10/23 08:08:16] root INFO: save model in ./output/hwdb\latest
[2021/10/23 08:08:16] root INFO: save model in ./output/hwdb\iter_epoch_195
[2021/10/23 08:08:16] root INFO: Initialize indexs of datasets:['./dataset2/train.txt']
[2021/10/23 08:08:17] root INFO: epoch: [196/200], iter: 1170, lr: 0.000070, loss: 0.370039, acc: 0.660156, norm_edit_dis: 0.660156, reader_cost: 0.09959 s, batch_cost: 0.15555 s, samples: 128, ips: 164.57493
[2021/10/23 08:08:20] root INFO: epoch: [196/200], iter: 1175, lr: 0.000068, loss: 0.363119, acc: 0.667969, norm_edit_dis: 0.667969, reader_cost: 0.35801 s, batch_cost: 0.63384 s, samples: 640, ips: 201.94361
[2021/10/23 08:08:21] root INFO: save model in ./output/hwdb\latest
[2021/10/23 08:08:21] root INFO: Initialize indexs of datasets:['./dataset2/train.txt']
[2021/10/23 08:08:25] root INFO: epoch: [197/200], iter: 1180, lr: 0.000065, loss: 0.373268, acc: 0.660156, norm_edit_dis: 0.660156, reader_cost: 0.41739 s, batch_cost: 0.69648 s, samples: 640, ips: 183.78153
[2021/10/23 08:08:26] root INFO: save model in ./output/hwdb\latest
[2021/10/23 08:08:26] root INFO: Initialize indexs of datasets:['./dataset2/train.txt']
[2021/10/23 08:08:30] root INFO: epoch: [198/200], iter: 1185, lr: 0.000062, loss: 0.366244, acc: 0.671875, norm_edit_dis: 0.671875, reader_cost: 0.31363 s, batch_cost: 0.53789 s, samples: 512, ips: 190.37307
[2021/10/23 08:08:32] root INFO: save model in ./output/hwdb\latest
[2021/10/23 08:08:32] root INFO: Initialize indexs of datasets:['./dataset2/train.txt']
[2021/10/23 08:08:34] root INFO: epoch: [199/200], iter: 1190, lr: 0.000059, loss: 0.358880, acc: 0.671875, norm_edit_dis: 0.671875, reader_cost: 0.25921 s, batch_cost: 0.42857 s, samples: 384, ips: 179.20252
[2021/10/23 08:08:37] root INFO: save model in ./output/hwdb\latest
[2021/10/23 08:08:37] root INFO: Initialize indexs of datasets:['./dataset2/train.txt']
[2021/10/23 08:08:39] root INFO: epoch: [200/200], iter: 1195, lr: 0.000057, loss: 0.368813, acc: 0.675781, norm_edit_dis: 0.675781, reader_cost: 0.19179 s, batch_cost: 0.30546 s, samples: 256, ips: 167.61677
[2021/10/23 08:08:43] root INFO: save model in ./output/hwdb\latest
[2021/10/23 08:08:43] root INFO: save model in ./output/hwdb\iter_epoch_200
[2021/10/23 08:08:43] root INFO: best metric, acc: 0
